src/components/restaurant/card/Body.tsx

```diff
@@ -37,21 +36,10 @@ function Section({ title, details }: { title: string; details: string }): ReactE
 }
 
 export default function Body({
-  address,
-  openingHours,
-  phoneNumber,
-}: Pick<Restaurant, 'address' | 'openingHours' | 'phoneNumber'>): ReactElement {
-  function formatHours(hours: Restaurant['openingHours']): string {
-    return `Sun: ${hours.sunday}
-      Mon: ${hours.monday}
-      Tue: ${hours.tuesday}
-      Wed: ${hours.wednesday}
-      Thu: ${hours.thursday}
-      Fri: ${hours.friday}
-      Sat: ${hours.saturday}
-    `;
-  }
-
+  formatted_address,
+  opening_hours,
+  formatted_phone_number,
```

Comment:
camelCase here

Reply:
I'm not too sure I can do all these camel Case's because they're coming from the `Place` interface

Reply:
Acvtually I think I might have actually restructured the Prop types in my other PRs to match a new style so no need to do this anymore!